doc/tutorial.rst

```diff
@@ -1131,7 +1131,7 @@ and ``.ml`` produced by ``atdgen``.
 GNU Make
 --------
 
-We provide `\`Atdgen.mk\` <https://github.com/mjambon/atdgen-make>`__, a generic
+We provide `Atdgen.mk <https://github.com/mjambon/atdgen-make>`__, a generic
```

Comment:
Should we just move this makefile into atd as well? It's quite easy to install into same shared directory with jbuild and it will let people reuse it with `-include $(opam config var share)/Atd.mk`.

Reply:
Yes, it's a good idea. I'm running out of time for this week. I'm creating an issue for it.

Reply:
-> https://github.com/mjambon/atd/issues/139